internal/restorer/fileswriter.go

```diff
@@ -4,98 +4,91 @@ import (
 	"os"
 	"sync"
 
-	"github.com/restic/restic/internal/debug"
-	"github.com/restic/restic/internal/errors"
+	"github.com/cespare/xxhash"
 )
 
-// Writes blobs to output files. Each file is written sequentially,
-// start to finish, but multiple files can be written to concurrently.
-// Implementation allows virtually unlimited number of logically open
-// files, but number of phisically open files will never exceed number
-// of concurrent writeToFile invocations plus cacheCap.
+// writes blobs to target files.
+// multiple files can be written to concurrently.
+// multiple blobs can be concurrently written to the same file.
```

Comment:
This might cause issues with FUSE based systems, especially FUSE mounted high-latency systems.

Reply:
@saikpr Can you please elaborate a bit on this? In what way can it cause issues, for what reason in FUSE?